doc/sdx-components.svg

```diff
@@ -0,0 +1,1106 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Probably want to make it clear that there are two receipt services reading off separate queues.

Also clarify that the receipting endpoints (and consumers thereof) are different to the (S)FTP buckets and the consumers of those.